.github/workflows/unity.yml

```diff
@@ -0,0 +1,57 @@
+name: unity
+
+on:
+  push:
+    branches:
+      - master
+      - 'support/*'
+  pull_request: {}
+
+jobs:
+  unity:
+    name: Unity build
+
+    strategy:
+      fail-fast: false
+      matrix:
+        onoff:
+          - ON
+          - OFF
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cancel previous jobs for the same PR
+        if: "github.event_name == 'pull_request'"
+        uses: styfle/cancel-workflow-action@89f242ee29e10c53a841bfe71cc0ce7b2f065abc
+        with:
+          workflow_id: deb.yml,docker.yml,raspbian.yml,rpm.yml,unity.yml,windows.yml
+          access_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We're reaching a point where I'm wondering if it would be possible to include this block from another file (in the same repo, making a separate action repo would be overkill).

Reply:
Please have a look at https://docs.github.com/en/actions/using-workflows/reusing-workflows, this should avoid having to duplicate the exact version of `styfle/cancel-workflow-action` and `workflow_id` in all workflow files.

Reply:
Should be solved with #9636